### ISSUE TITLE
OpenSHMEM+UCX local protection error under stress 

### ISSUE THREAD

**Issue by shamisp** [openucx/ucx#857]
```
 mpirun -np 16 -npernode 8 -H s-3,s-2  --mca pml ucx --mca spml ucx -x OMPI_MCA_mpi_add_procs_cutoff=10000000  -x UCX_TLS=rc,ud,mm -x MLX5_CQE_SIZE=64  gups-shmem/gups
Running on 16 processors (PowerofTwo)
Total Main table size = 2^18 = 262144 words
PE Main table size = 2^18 = 16384 words/PE
Default number of updates (RECOMMENDED) = 1048576
Real time used = 2.180833 seconds
0.000480814 Billion(10^9) Updates    per second [GUP/s]
0.000030051 Billion(10^9) Updates/PE per second [GUP/s]
mlx5: softiron-2: got completion with error:
00000000 00000000 00000000 00000000
00000000 00000000 00000000 00000000
00000005 00000000 00000000 00000000
00000000 01006102 0800012e 1f3a2cd2
[1467152031.902965] [softiron-2:2132 :0] rc_verbs_iface.c:47   UCX  ERROR Send completion with error: local QP operation error
[1467152048.497975] [softiron-3:3087 :0] rc_verbs_iface.c:47   UCX  ERROR Send completion with error: transport retry counter exceeded
[1467152048.498544] [softiron-3:3087 :1]      ib_device.c:132  UCX  WARN  IB Async event on mlx5_0: SRQ-attached QP 0xa18 was flushed
```

**Comment by yosefe** [comment]
@shamisp does this happen during exit flow?

**Comment by shamisp** [comment]
Not really. In the middle of the run.

**Comment by yosefe** [comment]
@alex-mikheev can we close it now?

**Comment by alex-mikheev** [comment]
i don't think so. At the moment I am experimenting with shmem only using single transport (rc or dc)

**Comment by yosefe** [comment]
@shamisp it should be fixed by #969

**Comment by shamisp** [comment]
let's close for now
